core/trino-main/src/test/java/io/trino/sql/planner/TestDynamicFilter.java

```diff
@@ -249,6 +289,22 @@ public void testJoin()
                                                 tableScan("lineitem", ImmutableMap.of("LINEITEM_OK", "orderkey")))))));
     }
 
+    @Test
+    public void testJoinConditionOrderReversed()
```

Comment:
`testInnerJoinWithConditionReversed`

Also, isn't there really a test here that has condition reversed for inner join?

Reply:
I don't see it in this class but I suspect it's tested somehow.
I do feel like having this simple case explicitly tested makes sense but it's actually not related to this change so I can remove it.